modules/dynamic-plug-in-sdk-extensions.adoc

```diff
@@ -1025,14 +1095,14 @@ events
 [discrete]
 == `console.topology/adapter/build`
 
-`BuildAdapter` contributes an adapter to adapt element to data that can be used by the `Build` component.
+`BuildAdapter` contributes an adapter to adapt element to data that can be used by the Pod component.
```

Comment:
Is this change correct?

Reply:
So like this?
```suggestion
`BuildAdapter` contributes an adapter to adapt element to data that can be used by the Build configuration.
```
and same for the one below?